File: Views/src/components/out-sourced/collect-deliver/dto/billing-from-dto.ts
import { CustomerDto } from "../../../../components/main/customer/components/commons-components/dtos/customer-dto";
import { PartnerDto } from "../../../../components/main/partner/dtos/partner-dto";
import { RootBase } from "../../../../shared/entities-dtos/root-base";



export class BillingFromDto extends RootBase {
  partnerId: number | null;
  partner: PartnerDto | null;

  customerId: number | null;
  customer: CustomerDto | null;

  base: boolean;
}

File: Application/Services/Operations/Outsourced/Dtos/BillingFromDto.cs
using Application.Services.Operations.Main.Customers.Dtos;
using Application.Services.Operations.Main.Partners.Dtos;
using Application.Services.Shared.Dtos;

namespace Application.Services.Operations.Outsourced.Dtos
{
    public class BillingFromDto : RootBaseDto
    {
        public int? PartnerId { get; set; }
        public PartnerDto Partner { get; set; }

        public int? CustomerId { get; set; }
        public CustomerDto Customer { get; set; }

        public bool Base { get; set; }
    }
}
